apps/passport-server/migrations/76_manual_ticket_checkin_type.sql

```diff
@@ -0,0 +1,2 @@
+alter table generic_issuance_checkins
+add column checkin_type VARCHAR DEFAULT NULL;
```

Comment:
note that there might be some ethberlin check-ins on staging-richard that are intended to be for swag - not a big deal but reminder to check those and edit